apps/web/components/App.tsx

```diff
@@ -83,7 +97,7 @@ export default function App({
               </div>
 
               <div className="mt-4 sm:mt-0 sm:text-right">
-                {isGlobal ? (
+                {data && !isLoading && isSuccess && credentialId && variant !== "calendar" ? (
```

Comment:
We want to let users connect multiple calendars so we shouldn't disable the install button